### PR TITLE
Few fixes for Salesforce

### PR DESCRIPTION
**Pull request by rlincoln24**
A few small fixes for salesforce lightning elements that are not properly inverted as well as a couple of other small tweaks in displays on the page.

Hiding a background image and hiding some text on dashboard widgets that add excessive bloat to the page.

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -3584,6 +3584,65 @@ IGNORE IMAGE ANALYSIS
 .is2d .queen.black
 .is2d .rook.black
 
+================================
+
+lightning.force.com
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:3589]: Invalid URL

**Review by Gusted — CHANGES_REQUESTED** [review]

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (rlincoln24) [reply src/config/dynamic-theme-fixes.config:3589]: It's intranet page you won't be able to access it unfortunately.

**Review by Gusted — CHANGES_REQUESTED** [review]
After some formatting the code it should look like
```CSS
lightning.force.com

CSS
.slds-brand-band,
.slds-brand-band:after,
.slds-brand-band_cover,
.slds-brand-band_medium,
.slds-card,
.slds-page-header,
.slds-clearfix {
    background-color: var(--darkreader-neutral-background) !important;
}

.slds-brand-band:before {
    background-image: none !important;
}

.gridFooter {
    visibility: hidden !important;
}

.slds-button_neutral,
.slds-button--neutral {
    color: rgb(20, 150, 20) !important;
}

.slds-context-bar {
    border-bottom-color: rgb(150, 150, 150) !important;
} 

================================
```

**Comment by rlincoln24** [comment]
Gusted I am still really new with Git (this is really my first time ever contributing to anything on Git) I am not sure what happens next with this. Is there something I can read somewhere that explains to me more what happens now, or if you could maybe PM me if your so inclined? Thanks!

**Comment by Gusted** [comment]
Well, each GIT repo has it's own workflow of how contribution goes. You can find specific instructions on readme.MD about contributing.

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -3584,6 +3584,39 @@ IGNORE IMAGE ANALYSIS
...
+.slds-button_neutral,
+.slds-button--neutral {
+    color: rgb(20, 150, 20) !important;
+}
+
+.slds-context-bar {
+    border-bottom-color: rgb(150, 150, 150) !important;
+} 
+
+
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:3619]: Remove the extra line here to pass the test.

**Review by rlincoln24 — COMMENTED** [review]
Looks good, thanks Gusted!

**Pull request merged**